### ISSUE TITLE
Błąd z sethome

### ISSUE THREAD

**Issue by RikoDEV** [xVacuum/qEssentialsReloaded#12]
Dzisiaj paru graczy napisało do mnie że nie działa im poprawnie sethome. Gdy wpisze się **/sethome lol** i potem **/home lol** to pisze że nie ma takiego domu:
https://s31.postimg.org/pzfzu2djv/bug.png

Prosiłbym o szybkie naprawienie tego błędu

**Comment by xVacuum** [comment]
To znaczy potrzebuje więcej informacji, dokładne logi, najlepiej zdjęcie z bazy danych (jak SQLite to poproszę o screen z tabelki `homes` w `SQLite Browser` (wpisz w google i będzie programik)

**Comment by xVacuum** [comment]
Tak, jest ogólne uprawnienie `.teleport` nie jest to bug, a zmiana nie jest zbyt dobra - bo później będzie płacz że nie wiedzą jaka jest nowe.

**Comment by RikoDEV** [comment]
@xVacuum 
**Log:** 
```
[08:33:13] [Server thread/INFO]: PolskiTestowy issued server command: /drop
[08:33:44] [Server thread/INFO]: PolskiTestowy issued server command: /drop
[08:33:54] [Server thread/INFO]: PolskiTestowy issued server command: /sethome
[08:34:00] [Server thread/INFO]: PolskiTestowy issued server command: /sethome lol
[08:34:03] [Server thread/INFO]: PolskiTestowy issued server command: /home lol
[08:34:05] [Server thread/INFO]: PolskiTestowy issued server command: /home lol
[08:34:10] [Server thread/INFO]: PolskiTestowy issued server command: /home
```
nic nie sypie jak widać

**SS z tego programiku:**
https://s32.postimg.org/qbi4emi7p/sql.png

**Comment by xVacuum** [comment]
Nie nie, musisz mi podać zawartości tych tabelek. Poszukaj, gdzieś tam jest zakładka.

**Comment by RikoDEV** [comment]
Zawartość tabelki home:
https://s32.postimg.org/qyijoxl4l/baza.png

próbowałem z restartami oraz usuwaniu tych tabelek ale to nie pomaga.

**Comment by xVacuum** [comment]
Bardzo dziwne... i tutaj nie mam pomysłu o co może chodzić kompletnie ;_; jeszcze gdyby się nie zapisywało to ok, wykrzaczył się system zapisu, gdyby działało po restarcie można by wnioskować że problem z ładowaniem, a tutaj kurwa nie wiem ._.

może jeszcze mi coś do głowy wpadnie, na razie nie mam kompletnie pomysłu. Popróbuj tam resetowac etc. nie mam pojęcia

**Comment by RikoDEV** [comment]
Spk ;/

**Comment by xVacuum** [comment]
Ok, teraz widzę że rzeczywiście jest problem a stało się to po ostatnich updatach (bo działało).

Aczkolwiek u mnie objawia się tym, że dopiero po reloadzie dany `home` działa. Będę drążył, na razie sprawa wydaje się podejrzana, bo w kodzie nie ma nic "złego". No nic, zobaczymy.
